packages/blockchain-extension/package.json

```diff
@@ -1504,7 +1505,8 @@
             ".pid",
             ".port",
             ".feature",
-            ".xml"
+            ".xml",
+            ".ts"
```

Comment:
we shouldn't be ignoring .ts files in the licence checker as all typescript files should have a licence

Reply:
indeed, modified license format (it's the same license)